app/src/main/java/org/schabi/newpipe/util/NavigationHelper.java

```diff
@@ -111,6 +111,17 @@ private NavigationHelper() {
                 .putExtra(Player.SELECT_ON_APPEND, selectOnAppend);
     }
 
+    @NonNull
+    public static <T> Intent getPlayerNextIntent(@NonNull final Context context,
```

Comment:
I'd call this `getPlayerEnqueueNextIntent`

Reply:
done